documentation/ipi-install/modules/ipi-install-configuring-the-install-config-file.adoc

```diff
@@ -29,23 +29,3 @@ endif::[]
 [kni@provisioner ~]$ mkdir ~/clusterconfigs
 [kni@provisioner ~]$ cp install-config.yaml ~/clusterconfigs
 ----
-
-. Ensure all bare metal nodes are powered off prior to installing the {product-title} cluster.
-+
-[source,bash]
-----
-[kni@provisioner ~]$ ipmitool -I lanplus -U <user> -P <password> -H <management-server-ip> power off
-----
```

Comment:
While this should go in troubleshooting, We should also include this in the depoying doc itself. I'm wondering if it should be a section before chapter 11 (deploying cluster), or if it should be an instruction inside chapter 11.

Reply:
![imagen](https://user-images.githubusercontent.com/312463/83643902-ec916480-a5b0-11ea-8aa0-d15199ad0954.png)

Reply:
what about like this?

Reply:
Added